### ISSUE TITLE
Data Difference 

### ISSUE THREAD

**Issue by linzhang-allan** [dms-vep/SARS-CoV-2_XBB.1.5_RBD_DMS#15]
Hello,
I noticed that the file 'sera_343C_mediumACE2_mut_effect.csv' in your repository contains the same columns as 'sera_343C_mediumACE2_mut_icXX.csv', such as 'Lib2-230815-sera-343C_mediumACE2'. However, the values between the two files differ.
After looking through the paper "Spike deep mutational scanning helps predict success of SARS-CoV-2 clades," it seems that these values might be derived using the following equation:
![1](https://github.com/user-attachments/assets/011b5802-811a-4d91-ab40-24079156c825)

I would appreciate it if you could explain the difference.

**Comment by Bernadetadad** [comment]
`sera_343C_mediumACE2_mut_effect.csv` contains escape values for each mutation in the library, which are the ϐ parameters in the biophysical model we are fitting to our variant selection data, this is described in the documentation [here](https://jbloomlab.github.io/polyclonal/biophysical_model.html). 
We can also convert those escape values into inhibitory concentration values (in this case IC90, conversion is described in the same documentation page), which I think most people are more familiar with and that's what `sera_343C_mediumACE2_mut_icXX.csv` file contains and why the values between the two files differ. 
Does this help?

**Comment by linzhang-allan** [comment]
Thanks, it does help a lot. However, I still have some doubts. It seems that the IC90 values for each mutant in the paper include both experimental measurements and model predictions, right? I'm interested in collecting the experimental IC90 data. If you could provide details on how many SARS-CoV-2 backbone DMS variants your lab measured IC90 for through wet-lab experiments, I would be very grateful.

**Comment by linzhang-allan** [comment]
By the way, I'm curious whether the DMS measurements of escape in Figure 4d of the paper ‘Spike deep mutational scanning helps predict success of SARS-CoV-2 clades’ refer to the β parameters or the biophysical outcomes?

**Comment by Bernadetadad** [comment]
> Thanks, it does help a lot. However, I still have some doubts. It seems that the IC90 values for each mutant in the paper include both experimental measurements and model predictions, right? I'm interested in collecting the experimental IC90 data. If you could provide details on how many SARS-CoV-2 backbone DMS variants your lab measured IC90 for through wet-lab experiments, I would be very grateful.

There's validation notebook linked in methods section where you can find all that data. Neutralization data is [here](https://github.com/jbloomlab/SARS-CoV-2-XBB.1.5_Spike_DMS_validations/blob/main/virus_neutralization.ipynb)

**Comment by Bernadetadad** [comment]
> By the way, I'm curious whether the DMS measurements of escape in Figure 4d of the paper ‘Spike deep mutational scanning helps predict success of SARS-CoV-2 clades’ refer to the β parameters or the biophysical outcomes?

It's mutation-level escape values, so β. you can see that in the same validation notebook I posted above.

**Comment by linzhang-allan** [comment]
Thanks for your explaination. I wonder how to calculate the escape values in the file "sera_343C_mediumACE2_mut_effect.csv" .

**Comment by linzhang-allan** [comment]
Because some negative values appear in this file, different the calculation method described in the paper "Complete Mapping of Mutations to the SARS-CoV-2 Spike Receptor-Binding Domain that Escape Antibody Recognition".

**Comment by Bernadetadad** [comment]
To get escape values for each mutation we fit the previously mentioned polyclonal model to variant escape probabilities, you can see this done in [the following notebook](https://dms-vep.org/SARS-CoV-2_XBB.1.5_RBD_DMS/notebooks/fit_escape_antibody_escape_Lib1-230815-sera-343C_mediumACE2.html) for one of the libraries. The same is done for the other library and results are averaged as shown [here](https://dms-vep.org/SARS-CoV-2_XBB.1.5_RBD_DMS/notebooks/avg_escape_antibody_escape_sera_343C_mediumACE2.html). 
The paper you refer to above was looking at changes in binding (not neutralization) using yeast DMS system, which required quite different analysis method @jbloom can expand on that.

**Comment by jbloom** [comment]
Yes, as @Bernadetadad notes the paper you refer to above uses yeast display whereas this repository is using a newer pseudovirus method.
